x-pack/qa/rolling-upgrade-multi-cluster/src/test/java/org/elasticsearch/upgrades/CcrRollingUpgradeIT.java

```diff
@@ -23,13 +23,7 @@
 import static org.hamcrest.Matchers.equalTo;
 
 public class CcrRollingUpgradeIT extends AbstractMultiClusterUpgradeTestCase {
-
-    // TODO: Remove this empty test
-    public void testEmpty() {
-
-    }
-
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/59625")
```

Comment:
Do we need to enable only this test or all of them? There's a couple of `@AwaitsFix` annotations left in `CcrRollingUpgradeIT` all pointing to the same Github issue.